utils/cleaners.py

```diff
@@ -358,6 +356,39 @@ def cuckoo_clean_lower_score(args):
     resolver_pool.map(lambda tid: delete_data(tid), id_arr)
 
 
+def tmp_clean_before_day(args):
+    """Clean up tmp folder
+    It deletes all items in tmp folder before now - days.
+    """
+    if not args.delete_tmp_items_older_than_days:
+        log.info("No days argument provided bailing")
```

Comment:
I know it's the same like the other comments, but lets write a nicer info message